packages/stemmers/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@orama/stemmers",
+  "version": "1.0.0-beta.15",
+  "type": "module",
+  "description": "Stemmers for Orama",
+  "sideEffects": false,
+  "main": "./dist/index.cjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
```

Comment:
That means that we're exporting all the stemmers together, right? Can you please create an export for EACH stemmer here?

Reply:
We already have it: it's the * export on the next entry.
Are you saying we should not have an index at all?